recirq/time_crystals/dtcsimulation_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 Google
```

Comment:
Are these tests sufficient coverage?
I don't see a test for `simulate_dtc_circuit_list` for instance.